.appveyor.yml

```diff
@@ -11,25 +11,19 @@ environment:
   QT_DOWNLOAD_HASH: '9a8c6eb20967873785057fdcd329a657c7f922b0af08c5fde105cc597dd37e21'
   QT_LOCAL_PATH: 'C:\Qt5.9.8_x64_static_vs2019'
   VCPKG_INSTALL_PATH: 'C:\tools\vcpkg\installed'
-  VCPKG_COMMIT_ID: 'f3f329a048eaff759c1992c458f2e12351486bc7'
+  VCPKG_COMMIT_ID: '13590753fec479c5b0a3d48dd553dde8d49615fc'
```

Comment:
Why the latest [release](https://github.com/microsoft/vcpkg/releases), i.e., https://github.com/microsoft/vcpkg/commit/56fffbe49dfb4dd8fae0940f272c5fd2b86be991, is not used?

Reply:
No good reason. I've always built from the vcpkg `master` branch, checked that the dependencies work and used that commit. The `vcpkg` repo is a bit different to code only repos in that commits are made for the library port configs as well as the source code use to build the `vcpkg` executable.